x-pack/plugins/security_solution/public/resolver/types.ts

```diff
@@ -288,10 +288,50 @@ export type ProcessWidths = Map<ResolverEvent, number>;
  * Map of ProcessEvents (representing process nodes) to their positions. Calculated by `processPositions`
  */
 export type ProcessPositions = Map<ResolverEvent, Vector2>;
+
+export type DurationTypes =
+  | 'millisecond'
```

Comment:
:grey_question:  I honestly don't know what you do about i18n'ing types like this. It's something I was struggling with a while back.

Reply:
I think these are the discrete set of values returned. Seems ok. If there were a ton more you could give up and go `string` without harming anything. ICU messages themselves aren't typed so they won't care.